api/src/main/java/org/pmiops/workbench/db/model/NotebookCohortCache.java

```diff
@@ -0,0 +1,82 @@
+package org.pmiops.workbench.db.model;
+
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.springframework.data.annotation.ReadOnlyProperty;
+
+import javax.persistence.AttributeOverride;
+import javax.persistence.AttributeOverrides;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "notebook_cohort_cache")
+public class NotebookCohortCache {
```

Comment:
It bothers me a bit that the model/table name is "cache", when really these are entries in a cache. Some alternative naming ideas; in particular it makes some of the service code look a bit off:

- `UserRecentResource`
- `NotebookCohortCacheEntry`
- `UserCachedResource`

Reply:
+1. I like `UserRecentResource` the best of these options